docs/apache-airflow/dag-run.rst

```diff
@@ -19,6 +19,26 @@ DAG Runs
 =========
 A DAG Run is an object representing an instantiation of the DAG in time.
 
+DAG Run Status
```

Comment:
I think this one is very much needed. It was not obvious at all, when DagRun is considered "success" I think.

I am not entirely sure myself if that is the "right" description, but it does seem plausible and reflects what I observed too. Anyone has comments on that ?

Reply:
I agree. It lacks some foreword about what DAG Run status actually is. I will try to extend it a bit, but any ideas are welcome!

Reply:
Please take a look and say what do you think about it now.